pretriage.py

```diff
@@ -114,8 +114,8 @@ def get_team_list():
 
     now = datetime.now()
     for vacation in json.loads(vacation):
-        if datetime.fromisoformat(vacation['end']) > now:
-            if datetime.fromisoformat(vacation['start']) < now:
+        if datetime.strptime(vacation['end'], '%Y-%m-%d') > now:
```

Comment:
I assume we're okay with this crashing and burning if it's invalid e.g. with a trailing space?

```python
>>> from datetime import datetime
>>> datetime.strptime('2022-05-06 ', '%Y-%m-%d')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python3.10/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/lib64/python3.10/_strptime.py", line 352, in _strptime
    raise ValueError("unconverted data remains: %s" %
ValueError: unconverted data remains
```

Reply:
I would be okay tbh. We have loud alerts on Slack if the pretriage bot fails